aikau/src/main/resources/alfresco/renderers/Selector.js

```diff
@@ -144,7 +144,8 @@ define(["dojo/_base/declare",
                   var b = lang.getObject(this.itemKey, false, this.currentItem);
                   var match = ((a || a === 0) && a === b);
                   if (match) {
-                     this.select();
+                     domClass.add(this.selectorNode, "checked");
+                     domClass.remove(this.selectorNode, "unchecked");
```

Comment:
It is existing class and it's used elsewhere. The whole thing could do with some rework, all of which was out of scope of this issue.